alexberka/a-trivial-glance
# Deletion of responses, gameQuestions on question deletion + Styling

## Description
Fixed bug wherein gameQuestion instances were not deleted when a question was deleted
Fixed bug that added 'responses' node within a team object on name update

Added Loading graphic on game load

Cross-Site Styling
Simplified some classes
Updated category.json with colors to contrast with black category text

## Related Issue
NA

## Motivation and Context


## How Can This Be Tested?
Check that all gameQuestions and responses are removed when a question is deleted

Check that styling loads and doesn't break on load of operation

## Screenshots (if appropriate):

## Types of changes
